### PR TITLE
Prevent numbers from being formatted in scientific notation in forms

### PR DESCRIPTION
**Pull request by maurizi**
Connects to #2733

### CODE REVIEW

**Comment by jwalgran** [comment]
The tag is working well for the built in fields but I added a custom `Decimal` field and the view mode value is getting scientifically notated.

Looks good in the form

screen shot 2017-08-31 at 2 42 47 pm

Notated after save

screen shot 2017-08-31 at 2 42 06 pm

**Comment by maurizi** [comment]
> Custom decimal field values in the view mode are getting scientifically notated.
Fixed in (rebased) commit https://github.com/OpenTreeMap/otm-core/pull/3208/commits/537c391b84f51f672a1c8d49eb7729fa4fd897c8. I also fixed formatting for numbers in the edits sidebar, and made sure we don't show trailing zeros

File: opentreemap/treemap/util.py [hunk 1]
```diff
@@ -231,3 +231,10 @@ def make_udf_name_from_key(key):
 
 def make_udf_lookup_from_key(key):
     return 'udfs__{}'.format(key)
+
+
+def num_format(num):
+    if isinstance(num, float):
+        # Allow for up to 10 digits of precision, but strip trailing 0s
+        return '{0:.10f}'.format(num).rstrip('0')
```
Comment (jwalgran) [inline opentreemap/treemap/util.py]: For any "whole number" value like `123.0` this `rstrip` is leaving a trailing `.`

screen shot 2017-09-06 at 9 55 21 am
 



screen shot 2017-09-06 at 10 00 55 am

Comment (maurizi) [reply opentreemap/treemap/util.py]: Fixed in https://github.com/OpenTreeMap/otm-core/pull/3208/commits/7bdee5f995cedd6ee22b6f523e475697726045da

File: opentreemap/treemap/util.py [hunk 2]
```diff
@@ -231,3 +231,10 @@ def make_udf_name_from_key(key):
 
 def make_udf_lookup_from_key(key):
     return 'udfs__{}'.format(key)
+
+
+def num_format(num):
+    if isinstance(num, float):
+        # Allow for up to 10 digits of precision, but strip trailing '0' or '.'
+        return '{0:.10f}'.format(num).rstrip('0.')
```
[the remark this answers is not in the document]
Comment (maurizi) [reply opentreemap/treemap/util.py]: Sigh... sorry for all the back and forth. Fixed in https://github.com/OpenTreeMap/otm-core/pull/3208/commits/149dfeafd83db8fa047dcbeaf5afef674fd40522

**Pull request merged**